### DISCUSSION TITLE
Missing function in shared.py script

### DISCUSSION THREAD

**Discussion by ZacharySha** [WAM2layers/WAM2layers#191] [Q&A] [answered]
Hello:
         When i run that code "WAM2layers/WAM2layers/blob/main/src/wam2layers/preprocessing/ecearth.py", it reported an error and was missing a function. i cheacked the script, and the "get_stable_fluxes" function and the "get_vertical_transport" function are not defined in shared.py.
     How can I solve this problem? 
  thanks！

**Comment by Peter9192** [comment, marked as answer]
Hi @ZacharySha, thanks for reaching out!

That code is very outdated - we should probably remove it from the repo altogether before the official v3 release. The best way forward if you want to use EC-Earth today is to write a new preprocessing script for it, similar to the one for ERA5 (on pressure levels).
